### PR TITLE
Docs: Update Command Summary Table

### PR DESCRIPTION
**Pull request by Pughal77**
The Command Summary table now includes all commands that are available in AvengersAssemble

### CODE REVIEW

**Review by delishad21 — COMMENTED** [review]
LGTM. Lets keep our UG updated so our users can have an optimal usage experience.

**Review by danielleloh — COMMENTED** [review]
simple fix, looks good to me :)

File: docs/UserGuide.md [hunk 1]
```diff
@@ -918,18 +918,29 @@ If this issue occurs, transfer the CSV file you want to import to the same folde
...
+| **Add**                | `add n\|NAME p\|PHONE_NUMBER e\|EMAIL a\|ADDRESS [t\|TAG]… [m\|MATRICULATION_NUMBER] [s\|STUDIO] [r\|REFLECTION]​` <br><br>• e.g. `add n\|James Ho p\|22224444 e\|jamesho@example.com a\|123, Clementi Rd, 1234665 t\|friend t\|colleague m\|A1234567X` |
+| **Edit**               | `edit INDEX [n\|NAME] [p\|PHONE_NUMBER] [e\|EMAIL] [a\|ADDRESS] [t\|TAG]… [m\|MATRICULATION_NUMBER] [s\|STUDIO] [r\|REFLECTION]​` <br><br>• e.g.`edit 2 n\|James Lee e\|jameslee@example.com m\|A1234567X`                                              |
+| **Delete**             | `delete INDEX` <br><br>• e.g. `delete 3`                                                                                                                                                                                                                |
+| **DeleteShown**        | `deleteShown`                                                                                                                                                                                                                                           |
```
Comment (danielleloh) [inline docs/UserGuide.md:929]: DeleteShown should be Delete Shown _(Persons(?))_

Anyways, should not be join since it should be reader-friendly.

**Review by danielleloh — COMMENTED** [review]
other then the small typo, everything else looks good to me

File: docs/UserGuide.md (hunk 1 above)
Comment (Zer0Legion) [reply docs/UserGuide.md:929]: Agreed on this

Comment (Pughal77) [reply docs/UserGuide.md]: Thanks for pointing that out!

**Review by jayllo-c — COMMENTED** [review]
The changes now properly showcase all commands in the correct order! Great job on this!

**Pull request merged**